api_end_end_testing/src/test/java/tds/user/api/tests/BaseUri.java

```diff
@@ -1,31 +1,70 @@
 package tds.user.api.tests;
 
+import java.io.FileInputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.Properties;
+
+
+import static com.jayway.restassured.RestAssured.*;
+
+import com.jayway.restassured.RestAssured;
+
 import org.testng.annotations.BeforeClass;
 
 /**
- * Created by mjbarger on 9/6/16.
+ *
  */
 public abstract class BaseUri {
+    String accessToken = null;
     String authenticateURI = null;
     String userUri = null;
 
-    @BeforeClass
+    @BeforeClass(alwaysRun = true)
     public void init() {
+        // Get properties needed for authentication
+/*
+        String configFileLocation = System.getProperty("config");
 
+        if (configFileLocation == null || configFileLocation.isEmpty()) {
+            throw new IllegalArgumentException("config file location system property is required (mvn test -Dconfig=<fileLocation>");
+        }
+*/
+        String configFileLocation = "/Users/mjbarger/config.properties";
```

Comment:
My concern here is on line 33:  `String configFileLocation = "/Users/mjbarger/config.properties";`.  I do not have a `/Users/mjbarger` on my computer anywhere.  If I want to run the regression tests, I have to do one of two things:

* Create a `/Users/mjbarger` directory and put the `config.properties` file there
* Update this file to point it to where my version of the `config.properties` file is

In my opinion, neither of those are desirable.  I think it might be worth trying the following:
* Add a `resources` directory under `/src/test` at the same level as the `java` directory (so you will have `/src/test/java` and `/src/test/resources`
* Put the `config.properties` file in the resources directory.
** You might have to tell IntelliJ that the `resources` directory you created is in fact a "resources" - type directory.  I don't remember how to do that off the top of my head, but I know I've done it.

Once you do the steps above, the `resources` should be in the`classpath`, so all you'll have to do is say `FileInputStream("config.properties")` or something like that.  Maybe check out this link for a better explanation:  http://www.techbeamers.com/read-data-properties-file-using-testng-framework/